pkg/nvlib/info/resolver.go

```diff
@@ -62,3 +69,35 @@ func (p platformResolver) ResolvePlatform() Platform {
 		return PlatformUnknown
 	}
 }
+
+// getPlatformOverride checks the system for a platform override file.
+// This allows system administrators to force the detection of a specific
+// platform.
+//
+// The first non-empty and non-comment line (starting with #) in the file is
+// returned.
+//
+// Note that no checks are performed for a valid platform value.
+//
+// This function can be overridden for testing purposes.
+var getPlaformOverride = func() (string, string) {
+	platformOverrideFile, err := os.Open("/etc/nvidia-container-toolkit/platform-override")
```

Comment:
Should this be pulled out to a constant.
Independently, it feels a bit weird to bake in a path referencing the `nvidia-container-toolkit` inside nvlib.

Reply:
I agree, the path to file should come from the entity importing `go-nvlib`